ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -1997,19 +1997,6 @@ mod tests {
         assert_eq!(Type::Void.byte_size(), Some(0));
     }
 
-    #[test]
```

Comment:
Is the duplicate the test by the same name in the AOT IR? Note that that test operates on a different type by the same name.

Reply:
In `jit_ir.rs`!

Reply:
Ah yes, i see it. Thanks.